test/TensorFlowRuntime/control_flow_1.swift

```diff
@@ -1,10 +1,5 @@
-// RUN: %target-run-eager-swift %swift-tensorflow-test-run-extra-options
-
-// SR-9737: hanging tests in GPE GPU mode
```

Comment:
Todo: close [TF-28](https://bugs.swift.org/browse/TF-28) (previously SR-9737) if this PR is merged.